javascripts/discourse/components/category-list.js

```diff
@@ -0,0 +1,15 @@
+import Category from "discourse/models/category";
+import Component from "@glimmer/component";
+
+export default class CategoryList extends Component {
+  constructor() {
+    super(...arguments);
+    this.blockTitle =
+      this.args?.params?.title || I18n.t(themePrefix("category_list.heading"));
+
+    this.categoryList = settings.category_list.split("|");
```

Comment:
This component works a little differently from other components, in that each "block" gets its parameters from the same theme setting. 

Is it possible to avoid adding the `category_list` theme setting and have the list of categories be set in the JSON schema setting? We'd have to decide whether to use category IDs or category slugs in there, the interface would not be as friendly, but it would fit better with the data structure of the other blocks in the component. (For example, in `category-topics` we pass the category ID as a parameter.)